app/package.json

```diff
@@ -30,12 +30,19 @@
   },
   "dependencies": {
     "@babel/preset-react": "^7.0.0",
+    "@date-io/date-fns": "^1.1.0",
     "@material-ui/core": "^3.9.0",
     "@material-ui/icons": ">=3.0.0",
+    "date-fns": "^2.0.0-alpha.27",
+    "final-form": "^4.12.0",
+    "final-form-arrays": "^1.1.2",
     "gravatar": "^1.8.0",
     "history": "^4.7.2",
     "js-logger": "^1.6.0",
+    "material-ui-pickers": "^2.2.4",
     "prop-types": "^15.5.7",
+    "react-final-form": "^4.1.0",
+    "react-final-form-arrays": "2.0.1",
```

Comment:
Do you really need to duplicate `dependencies` from django-bananas.js here? Sounds strange to me. `peerDependencies` of course needs to be added here as `dependencies`, but other ones should not be needed…

Reply:
I don't think we ever install `django-bananas` when building the app image, which means we do not install its dependencies. So now that I removed those, build cannot be completed because of following errors:

```
ERROR in ./django-bananas/forms/FieldArray.js
Module not found: Error: in '/code/django-bananas/forms'
 @ ./django-bananas/forms/FieldArray.js 1:0-64 1:0-64
 @ ./django-bananas/forms/index.js
 @ ./src/pages/example/user/form.read.js
 @ ./src/pages lazy ^\.\/.*$ namespace object
 @ ./src/index.js
```

Reply:
Hmm, we should fix that at some point